documentation/src/main/wiki/HandlebarsKnot.md

```diff
@@ -58,7 +58,7 @@ Default configuration shipped with the verticle as `io.knotx.HandlebarsKnot.json
   "options": {
     "config": {
       "address": "knotx.knot.handlebars",
-      "templateDebug": true
```

Comment:
Why we don't support debugging any more?
Is that provided by another mechanism?

Reply:
Yes, a log level ;)

Reply:
But this was different thing.
This option wrapped resolved snippet with HTML comments, like
```html
<!-- start -->
processed snippet content
<!-- end -->
```